test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that PTS is created with appHmiType WEB_VIEW
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine App1 with WEB_VIEW HMI type is registered
+
+-- Sequence:
+-- 1. App2 register with NAVIGATION HMI type
+-- SDL does:
+--   a. PTU is triggered, SDL sends UPDATE_NEDDED to HMI
+--   b. PTS is created with AppHMIType WEB_VIEW for App1 and other mandatory pts
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local appSessionId2 = 2
+local appHMIType = "WEB_VIEW"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
```

Comment:
Please remove bracers
```suggestion
config.application1.registerAppInterfaceParams.appHMIType = appHMIType
```
Related to other places in this and other scripts

Reply:
@YarikMamykin  Done in  ([77cc64b](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/36/commits/77cc64bc938f0db605c2ba6024ef429f9d6ca497))